src/utils/live-tunnel.ts

```diff
@@ -92,16 +102,34 @@ const installTunnelClient = async function () {
   })
 }
 
-// @ts-expect-error TS(7031) FIXME: Binding element 'localPort' implicitly has an 'any... Remove this comment to see the full error message
-export const startLiveTunnel = async ({ localPort, netlifyApiToken, siteId, slug }) => {
+export const startLiveTunnel = async ({
+  localPort,
+  netlifyApiToken,
+  siteId,
+  slug,
+}: {
+  localPort: number
+  netlifyApiToken?: string | null | undefined
+  siteId?: string | undefined
+  slug: string
+}) => {
+  if (!siteId) {
+    console.error(
+      `${NETLIFYDEVERR} Error: no siteId defined, did you forget to run ${chalk.yellow(
+        'netlify init',
+      )} or ${chalk.yellow('netlify link')}?`,
+    )
+    process.exit(1)
```

Comment:
This is not a behavioural change. I just moved this from one method to another, but the result is the same